envoy/http/codec.h

```diff
@@ -389,6 +389,14 @@ class Stream {
    * @param the account to assign this stream.
    */
   virtual void setAccount(Buffer::BufferMemoryAccountSharedPtr account) PURE;
+
```

Comment:
Might be worth documenting what these bytes are (e.g. that they're wire bytes?)

Reply:
+1 please add interface comments here and elsewhere.